doc/content/gettingstarted/register.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Register Things"
```

Comment:
Let's also call it consistently devices or end devices, not things

Reply:
Well, this one give instructions for `devices` and `gateways`. So `Register gateways and devices` ?

Reply:
Yes